src/widgets/splits/Split.cpp

```diff
@@ -823,9 +823,11 @@ void Split::showSearch()
 
 void Split::reloadChannelAndSubscriberEmotes()
 {
-    getApp()->accounts->twitch.getCurrent()->loadEmotes();
+    auto currentUser = getApp()->accounts->twitch.getCurrent();
     auto channel = this->getChannel();
 
+    currentUser->loadEmotes();
```

Comment:
I don't see the point of this change ![pajaDent](https://static-cdn.jtvnw.net/emoticons/v2/301512758/default/dark/1.0)

Reply:
I think I needed `currentUser` property to call some other methods on it. Currently it can be reverted, you're correct. Thanks for noticing this!